components/ai_chat/core/browser/conversation_handler.cc

```diff
@@ -884,33 +887,49 @@ void ConversationHandler::PerformAssistantGeneration(
       base::BindOnce(&ConversationHandler::OnEngineCompletionComplete,
                      weak_ptr_factory_.GetWeakPtr());
 
-  bool should_refine_page_content =
-      features::IsPageContentRefineEnabled() &&
-      page_content.length() > GetMaxContentLengthForModel(GetCurrentModel()) &&
-      input != l10n_util::GetStringUTF8(IDS_AI_CHAT_QUESTION_SUMMARIZE_PAGE);
-  if (should_refine_page_content && associated_content_delegate_) {
-    DVLOG(2) << "Asking to refine content, which is of length: "
-             << page_content.length();
+  // Determine if page content refinement is necessary
+  const bool is_refinement_enabled = features::IsPageContentRefineEnabled();
```

Comment:
I feel we can just use `features::IsPageContentRefineEnabled()`. There's only a single read to `is_refinement_enabled`.